src/shrimps/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from datetime import datetime
+
+color_choices = [
+    ("pink", "pink"),
+    ("white", "white"),
+    ("green", "green"),
+    ("red", "red"),
+    ("purple", "purple"),
+    ("rainbow", "rainbow"),
+]
+
+
+class Shrimp(models.Model):
+    name = models.CharField(max_length=126)
+    birth_date = models.DateTimeField(default=datetime.now())
```

Comment:
here you should use django.utils.timezone.now 
becareful calling the function will set a constant rather than being evaluated each time an object is created.